cal/__init__.py

```diff
@@ -70,31 +70,39 @@ def home():
     return render_template('index.html')
 
 
-@app.route("/events/")
-def events():
+@app.route("/events/<week>")
+def events(week):
     now = dt.datetime.now()
-    events = Event.query.filter(Event.start > now) \
-        .filter(Event.start < now + dt.timedelta(weeks=1))
+    if week != "0":
+        now = dt.datetime.now() + dt.timedelta(weeks=int(week))
+    beginweek = now - dt.timedelta(days=now.weekday() + 2)
+    events = Event.query.filter(Event.start > beginweek) \
+        .filter(Event.start < beginweek + dt.timedelta(weeks=1))
 
     return jsonify(data=[event.to_json() for event in events.all()])
 
 
-@app.route("/users/")
-def users():
+@app.route("/users/<week>")
```

Comment:
Could you specify that `week` must be an `int`? The docs are [here](http://flask.pocoo.org/docs/0.10/api/#url-route-registrations).